infra/prod/_modules/functions/function_relying_party.tf

```diff
@@ -0,0 +1,85 @@
+locals {
+  relying_party_func_settings = {
+    for s in var.relying_party_func.app_settings :
+    s.name => s.key_vault_secret_name != null ? "@Microsoft.KeyVault(VaultName=${var.product}-kv;SecretName=${s.key_vault_secret_name})" : s.value
+  }
+}
+
+module "relying_party_func" {
+  source = "github.com/pagopa/terraform-azurerm-v3.git//function_app?ref=v7.72.2"
+
+  name                = format("%s-relying-party-func", var.project)
+  location            = var.location
+  resource_group_name = var.resource_group_name
+
+  health_check_path = "/health"
+
+  app_settings = local.relying_party_func_settings
+
+  node_version    = "18"
```

Comment:
20?

Reply:
In order to use node 20 we need `azurerm v3.96` at least https://registry.terraform.io/providers/hashicorp/azurerm/3.95.0/docs/resources/linux_function_app#node_version. It seems that we are still using some modules that specify azurerm version `<= 3.94.0` and so I am not able to update the version, I saw that the version of  all the modules we are using has been updated [here](https://github.com/pagopa/terraform-azurerm-v3/pull/262) so I do not understand why I'm getting this error trying to update
![image](https://github.com/pagopa/io-fims/assets/64153597/bdd07411-03b5-4459-95e5-7be850a7e660)

Reply:
@gunzip updating our terraform modules is out of scope right now

Reply:
right